live/tests/unit/components/qroc-solution-panel-test.js

```diff
@@ -6,12 +6,101 @@ describe('Unit | Component | qroc-solution-panel', function () {
 
   setupTest('component:qroc-solution-panel', {});
 
+  describe('#isResultOk', function () {
+
+    it('should return true when result is ok', function () {
+      // given
+      const answer = {
+        result: 'ok'
+      };
+      const component = this.subject();
+      component.set('answer', answer);
+      // when
+      const isResultOk = component.get('isResultOk');
+      // then
+      expect(isResultOk).to.be.true;
+    });
+
+    it('should return true when result is not ok', function () {
+      // given
+      const answer = {
+        result: 'ko'
+      };
+      const component = this.subject();
+      component.set('answer', answer);
+      // when
+      const isResultOk = component.get('isResultOk');
+      // then
+      expect(isResultOk).to.be.false;
+    });
+
+  });
+  describe('#isResultNotOk', function () {
+
+    it('should return true when result is ko', function () {
+      // given
+      const answer = {
+        result: 'ko'
+      };
+      const component = this.subject();
+      component.set('answer', answer);
+      // when
+      const isResultOk = component.get('isResultNotOk');
+      // then
+      expect(isResultOk).to.be.true;
+    });
+
+    it('should return true when result is not ko', function () {
+      // given
+      const answer = {
+        result: 'ok'
+      };
+      const component = this.subject();
+      component.set('answer', answer);
+      // when
+      const isResultOk = component.get('isResultNotOk');
+      // then
+      expect(isResultOk).to.be.false;
+    });
+
+  });
+
+  describe('#isResultWithoutAnswer', function () {
+
+    it('should return true when result is aband', function () {
+      // given
+      const answer = {
+        result: 'aband'
+      };
+      const component = this.subject();
+      component.set('answer', answer);
```

Comment:
Cette partie de code est valable pour les tests unitaires.
This pointe vers le contexte mais pas le composant.